src/resources/vue/components/model.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
rename the file to modal because the component is not a model

Reply:
fixed